src/js/burials/containers/ConfirmationPage.jsx

```diff
@@ -47,15 +52,44 @@ class ConfirmationPage extends React.Component {
         </p>
         <div className="inset">
           <h4>Burial Benefit Claim <span className="additional">(Form 21P-530)</span></h4>
-          <span>for {name.first} {name.middle} {name.last} {name.suffix}</span>
+          <span>for {claimantName.first} {claimantName.middle} {claimantName.last} {claimantName.suffix}</span>
 
           <ul className="claim-list">
             <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Deceased Veteran</strong><br/>
+              <span>{veteranName.first} {veteranName.middle} {veteranName.last} {veteranName.suffix}</span>
+            </li>
+            <li>
+              <strong>Benefits claimed</strong><br/>
+              {_.map(benefits, (isRequested, benefitName) => isRequested && <p>{benefitsLabels[benefitName]}</p>)}
+            </li>
+            <li>
+              <strong>Documents Uploaded</strong><br/>
+              {_.map(documents, (contents, documentName) => { // eslint-disable-line consistent-return
```

Comment:
I think it would be clearer to do something like this:

```js
{form.deathCertificate && <p>Death certificate: 1 file</p>}
{form.transportationReceipts &&
  <p>Transportation receipts: {form.transportationReceipts.length} {form.transportationReceipts.length > 1 ? 'receipts' : 'receipt'}</p>}
```